src/client/components/Navigation/Topnav.less

```diff
@@ -180,11 +183,15 @@ div[data-dir='rtl'] .Topnav {
 }
 
 .Topnav__search-dropdown-container.ant-select-dropdown {
-  top: 56px !important;
+  top: 45px !important;
   position: fixed;
   border-radius: 0;
-  border-bottom-left-radius: 4px;
-  border-bottom-right-radius: 4px;
+  border-bottom-left-radius: @border-radius-base;
+  border-bottom-right-radius: @border-radius-base;
+  border-right: @base-border;
+  border-left: @base-border;
+  border-bottom: @base-border;
+  box-shadow: none;
```

Comment:
Can you just add some `padding-top: 5px;` here to the input is not being cut with the autocomplete block.